src/utils/apv.ts

```diff
@@ -17,7 +17,7 @@ export function analyzeApv(token: string | AppProtocolVersionType): IApv {
 
 function analyzeApvFromToken(token: string): IApv {
   const [version, signer, signature, rawExtra] = token
-    .replace(".", "")
+    .replaceAll(".", "")
```

Comment:
APV seems to use the `'.'` character as a replacement for the `'/'` character.

So isn't it dangerous to remove all `'.'` characters?

For example, when `rawExtra` is `"...."` (means `"////"` in base64 standard), it will be `""`.

Reply:
Get safety confirm from @swen